css-shapes-2/Overview.bs

```diff
@@ -162,17 +172,32 @@ The ''shape()'' Function</h4>
 	Each command's starting point is the previous command's ending point.
 
 	<pre class=prod>
-		<<shape-command>> = <<move-command>> | <<line-command>> | <<hv-line-command>> |
-		                  <<curve-command>> | <<smooth-command>> | <<arc-command>> | close
+		<<shape-command>> = <<move-command>> | <<line-command>> | <<horizontal-line-command>> |
+						<<vertical-line-command>> | <<curve-command>> | <<smooth-command>> |
+						<<arc-command>> | close
```

Comment:
Not sure what indentation you're aiming to use but 4 consecutive tabs seems very suspicious.

I might suggest ordering these as
```suggestion
		<<shape-command>> = <<move-command>> | <<line-command>> | close
		                                        <<horizontal-line-command>> | <<vertical-line-command>> | 
		                                        <<curve-command>> | <<smooth-command>> | <<arc-command>>
```

so that related commands are on a single line.